examples/package.json

```diff
@@ -27,51 +21,30 @@
     "@cognite/sdk": "link:../viewer/node_modules/@cognite/sdk",
     "camera-controls": "^1.25.1",
     "dat.gui": "^0.7.7",
-    "geo-three": "^0.0.16",
     "hold-event": "^0.0.1",
-    "install": "^0.13.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-router-dom": "^5.2.0",
     "styled-components": "^5.2.0"
   },
   "devDependencies": {
-    "@actions/core": "^1.2.6",
-    "@actions/github": "^4.0.0",
     "@testing-library/jest-dom": "^5.11.5",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
     "@types/dat.gui": "^0.7.5",
-    "@types/jest": "^26.0.19",
-    "@types/jest-environment-puppeteer": "^4.4.1",
-    "@types/jest-image-snapshot": "^4.1.2",
-    "@types/node": "^14.14.5",
     "@types/react": "^16.9.54",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.6",
     "@types/styled-components": "^5.1.4",
     "@types/three": "0.141.0",
-    "@types/webpack-env": "^1.16.2",
-    "copy-webpack-plugin": "^6.2.1",
-    "cross-env": "^7.0.2",
     "css-loader": "^6.7.1",
-    "customize-cra": "^1.0.0",
     "dotenv-webpack": "^7.1.0",
     "html-webpack-plugin": "^5.5.0",
     "jest": "^28.1.1",
```

Comment:
Do we use jest for examples still, I thought jest was used only for visual test in examples

Reply:
Nice catch! Nope probably not, I'll remove it.